README.md

```diff
@@ -115,6 +117,41 @@ be available on their page, or you can visit http://localhost:6543/?embed=1&cont
 | React development server   | [`6543`](http://localhost:6543) |
 | Gunicorn for Django app    | [`8081`](http://localhost:8081) |
 
+
+### Parallel Development of OGR
+
+The Open Goods Registry is the successor to OAR and OGR will replace OAR when it
```

Comment:
Spelling changes:
"The Open Goods Registry is the successor to OAR, and OGR will replace OAR when it is launched. During the beta period, both applications will be deployed and developed in parallel. Features development for OAR also need to be included in OGR, but there are OGR-specific features and related architectural changes that are not appropriate to be included in OAR. Too support this parallel development process, there are 4 main branches in the repository:"

Reply:
Thanks. Updated in fixup 8dda5bc4